crates/protect-ffi/src/lib.rs

```diff
@@ -90,15 +133,17 @@ async fn new_client_inner() -> Result<Client, Error> {
     Ok(Client {
         cipher: Arc::new(cipher),
         zerokms,
+        encrypt_config: Arc::new(encrypt_config.to_config_map()),
     })
 }
 
 fn encrypt(mut cx: FunctionContext) -> JsResult<JsPromise> {
     let client = (**cx.argument::<JsBox<Client>>(0)?).clone();
     let plaintext = cx.argument::<JsString>(1)?.value(&mut cx);
     let column_name = cx.argument::<JsString>(2)?.value(&mut cx);
-    let lock_context = encryption_context_from_js_value(cx.argument_opt(3), &mut cx)?;
-    let service_token = service_token_from_js_value(cx.argument_opt(4), &mut cx)?;
+    let table_name = cx.argument::<JsString>(3)?.value(&mut cx);
+    let lock_context = encryption_context_from_js_value(cx.argument_opt(4), &mut cx)?;
+    let service_token = service_token_from_js_value(cx.argument_opt(5), &mut cx)?;
```

Comment:
Already discussed this with @calvinbrewer, but I want to update this function to use an opts object instead of positional args. I also want to do a pass at generally making the interface more consistent between the single and bulk versions of functions.